exercises/triangle/example.rb

```diff
@@ -1,44 +1,37 @@
-class TriangleError < RuntimeError
+module BookKeeping
+  VERSION = 1
 end
 
 class Triangle
-  attr_reader :a, :b, :c
-  def initialize(a, b, c)
-    @a = a
-    @b = b
-    @c = c
-  end
+  attr_reader :sides
 
-  def kind
-    fail TriangleError if illegal?
```

Comment:
Are we no longer raising on an illegal triangle?  I did not see a comment about this in the discussions here.